styles.css

```diff
@@ -41,11 +61,12 @@ nav {
 }
 
 .container {
-  background: url("/img/bg.jpeg");
+  background: url("/img/Header\ bg.png");
   background-size: 100% 100%;
   background-repeat: no-repeats;
```

Comment:
- [ ] Great work making some changes to the `background-image` of the _**top**_ section. However, I noticed that the `background-image` used is not the correct one and doesn't correlate with that of the Figma template used. Also, the positioning of the `background-image` is not uniform with that of the Figma template chosen. 👍 

### Your Design

![Screenshot3](https://user-images.githubusercontent.com/77106962/222587823-9b0bc990-f1e6-4b71-b1e9-bde5176ba7e4.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/222588011-473cd765-d061-417d-a381-51b0b94372b0.png)